tests/components/synology_dsm/test_config_flow.py

```diff
@@ -496,3 +498,23 @@ async def test_options_flow(hass: HomeAssistantType, service: MagicMock):
     assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
     assert config_entry.options[CONF_SCAN_INTERVAL] == 2
     assert config_entry.options[CONF_TIMEOUT] == 30
+
+
+async def test_services_registered(hass: HomeAssistantType):
```

Comment:
This test should not be part of the config flow tests. It should move to a `test_init.py` module.